### PR TITLE
Add initial support for generating triples.

### PR DESCRIPTION
**Pull request by keyurva**
* Triples are saved in a `triples.csv` in the output directory.
* Currently it only generates SV triples based on the SV column names.
   + SV descriptions and hierarchy support will be added next.
* Also, made `stats` into a module to eliminate the need for funky imports.

### CODE REVIEW

File: simple/sample/output/triples.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+subject_id,predicate,object_id,object_value
```
Comment (shifucun) [inline simple/sample/output/triples.csv:1]: Mixer now can only read from one "raw data" folder, so the triples csv would ideally be in the same folder as the observations.

**Review by shifucun — COMMENTED** [review]
Very nice clean up on the module and folder structure!

File: simple/sample/output/triples.csv (hunk 1 above)
Comment (keyurva) [reply simple/sample/output/triples.csv:1]: Triples csv is in the same folder as observations. Basically, the same folder where you read the observations to load the sqlite db.

Comment (shifucun) [reply simple/sample/output/triples.csv:1]: Wonder why the triples.csv here is not in the same folder of observations?

**Review by shifucun — APPROVED** [review]

File: simple/sample/output/triples.csv (hunk 1 above)
Comment (keyurva) [reply simple/sample/output/triples.csv:1]: It is in the same folder: https://screenshot.googleplex.com/8G86oAmfkmgU4ht

If you're implying a different folder, let me know.

**Pull request merged**